webview/src/plots/components/ZoomablePlotWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement, PropsWithChildren } from 'react'
+import Tooltip from '../../shared/components/tooltip/Tooltip'
+
+interface ZoomablePlotWrapperProps {
+  title?: string
+  id: string
+}
+
+export const ZoomablePlotWrapper: React.FC<
+  PropsWithChildren<ZoomablePlotWrapperProps>
+> = ({ title, id, children }) => {
+  const isTitleCut = title?.indexOf('…') === 0
+
+  return isTitleCut ? (
+    <Tooltip content={id} placement="top" interactive>
+      {children as ReactElement}
+    </Tooltip>
```

Comment:
The cursor changes to a grab icon when you hover over the tooltip: 

https://github.com/iterative/vscode-dvc/assets/43496356/d778d6a0-d8c7-46d0-b8da-f31fac51d704

Reply:
Fixed! Good catch!